code-reviews/recipes/JavaScript.md

```diff
@@ -2,27 +2,136 @@
 
 ## Style Guide
 
-We use [`prettier`](https://prettier.io/) to do code formatting for JavaScript. Using an automated code formatting tool like Prettier enforces a well accepted style guide that was collaboratively built by a wide range of companies including Microsoft, Facebook, and AirBnB. It is highly recommended that you use one of the native editor integrations and enable format on save. In addition, by default, we use the following overrides (shown here in the config format for VSCode) to standardize on single quotes, a four space drop, and to do ESLinting:
+[CSE](../../CSE.md) developers use [prettier](https://prettier.io/) to do code formatting for JavaScript.
 
+Using an automated code formatting tool like Prettier enforces a well accepted style guide that was collaboratively built by a wide range of companies including Microsoft, Facebook, and AirBnB.
+
+For higher level style guidance not covered by prettier, we follow the [AirBnB Style Guide](https://github.com/airbnb/javascript).
+
+## Linter
```

Comment:
Maybe this should be called linting instead

Reply:
Can you elaborate on your reasoning here?

Reply:
no specific reason just my personal preference. I was thinking of "Linting Guide"   Instead of Linter or Linters

Reply:
Cool, I think linter/s fits better with the theme of the other section titles, but let's see what @debugerr thinks

Reply:
I like linting because "linting" is something that is done by a "linter" (but can also be done manually), so in terms of categorizing it, "linting" is broader.

Reply:
Fixed

Reply:
Both work for me. I agree linting is a bit broader.